producers.md

```diff
@@ -485,10 +485,10 @@ testing frameworks and supports logging TAP.
 **[SimpleTest TAP reporter](http://digitalsandwich.com/Updated-Simpletest+Apache-Test/)**
 is a TAP producer for [SimpleTest](http://www.simpletest.org/).
 
-**[SnapTest](http://www.snaptest.net/)** is a powerful unit testing framework
-for PHP 5+, leveraging PHP's unique runtime language to simplify the unit
-test process without sacrificing the agility tests provide. It has a built-in
-TAP reporter.
+
+
+
+
```

Comment:
Can you delete the whitespace, please?

Reply:
Somehow backspace didn't work from the browser editor on my mobile, thanks for removing them.